src/backends/multicore/mechanism.cpp

```diff
@@ -27,6 +27,9 @@ namespace multicore {
 using util::make_range;
 using util::value_by_key;
 
+static constexpr unsigned simd_width_ = S::simd_abi::native_width<fvm_value_type>::value;
```

Comment:
We use the trailing underscore `name_` for member variables of classes.
The following should be sufficient:
```
constexpr unsigned simd_width = S::simd_abi::native_width<fvm_value_type>::value;
```
Note that you don't need `static` to restrict the visibility of a `constexpr` variable in this context.